tests/unit/dag/test_graph.py

```diff
@@ -95,3 +98,42 @@ def test_subgraph_with_summed_subgraphs():
     assert post_len == pre_len
     assert iter_len == post_len
     assert iter_len == pre_len
+
+
+def test_selector_concat_parents():
```

Comment:
It looks like this tests that an overlapping column name prefers the version from the right-hand side. Not quite sure how to distill that into a good test name though...`test_concat_prefers_rhs`?